core/tests/gae_suite.py

```diff
@@ -45,6 +45,7 @@
     os.path.join(OPPIA_TOOLS_DIR, 'Pillow-6.2.2'),
     os.path.join(OPPIA_TOOLS_DIR, 'psutil-5.7.0'),
     os.path.join(OPPIA_TOOLS_DIR, 'PyGithub-1.45'),
+    os.path.join(OPPIA_TOOLS_DIR, 'pylatexenc-2.5'),
```

Comment:
Wrong dir. I thought I mentioned this in a previous review?

Reply:
Changed.